kobo/apps/organizations/templates/emails/accepted_invite.txt

```diff
@@ -0,0 +1,10 @@
+{% load i18n %}
+{% trans "Dear" %} {{ sender_username }},
+
+{% blocktrans %}{{ recipient_username }} ({{ recipient_email }}) has accepted your request to join {{ organization_name }} organization.{% endblocktrans %}
+
+{% trans "All projects, submissions, data storage, transcription and translation usage for their projects will be transferred to you." %}
+
+{% trans "Note: You will continue to have permissions to manage these projects until the user permissions are changed." %}
```

Comment:
I don't think this sentence should be here since that's the sender who receives this message.
(Please fix HTML template too)